thenewboston_node/business_logic/models/node.py

```diff
@@ -26,17 +26,24 @@ class Node(BaseDataclass):
     fee_account: Optional[hexstr] = field(
         default=None, metadata={'example_value': '7a5dc06babda703a7d2d8ea18d3309a0c5e6830a25bac03e69633d283244e001'}
     )
+    role: Optional[int] = field(default=None, metadata={'example_value': 3})
```

Comment:
the attributes added here are supposed to be stored on the blockchain, signed, etc. `role` is already stored in the other way (as PV schedule), storing `is_online` does not make sense since it is changing, the same about `last_block_number`